backend/danswer/main.py

```diff
@@ -341,6 +370,9 @@ async def lifespan(app: FastAPI) -> AsyncGenerator:
 
         translate_saved_search_settings(db_session)
 
+        # update multipass indexing setting based on GPU availability
+        update_default_multipass_indexing(db_session)
```

Comment:
I would place this later (after the embedding warm up setup) because that step has backoff and retries. Otherwise this call to check GPU could run before the model server is actually up

Reply:
note that the call later on checks for the inference server, not indexing